dd-sdk-android/src/main/kotlin/com/datadog/android/DatadogInterceptor.kt

```diff
@@ -65,13 +67,16 @@ import okhttp3.Response
  * continue to dispatch RUM Resource events for each request without applying any host filtering.
  * @param tracedRequestListener which listens on the intercepted [okhttp3.Request] and offers
  * the possibility to modify the created [io.opentracing.Span].
- *
+ * @param rumResourceAttributesProvider which listens on the intercepted [okhttp3.Request]
+ * and offers the possibility to add custom attributes to the RUM resource events.
  */
 open class DatadogInterceptor
 internal constructor(
     tracedHosts: List<String>,
     tracedRequestListener: TracedRequestListener,
     firstPartyHostDetector: FirstPartyHostDetector,
+    private val rumResourceAttributesProvider: RumResourceAttributesProvider =
```

Comment:
it is not last in the argument list to keep lambda last to still have `ctor() { lambda }` syntax